README.md

```diff
@@ -51,67 +43,48 @@ if (scope != null) {
 
 ### Starting a new Span
 
-The common case starts an `Scope` that's automatically registered for intra-process propagation via `ScopeManager`. The best practice is to use a try-with-resources pattern which handles Exceptions and early returns:
+The common case starts a `Scope` that's automatically registered for intra-process propagation via `ScopeManager`.
 
-```java
-io.opentracing.Tracer tracer = ...;
-...
-try (Scope scope = tracer.buildSpan("someWork").startActive()) {
-    // Do things.
-    //
-    // If we create async work, `scope.span()` allows us to pass the `Span` along as well.
-}
-```
-
-The above is semantically equivalent to the more explicit try-finally version:
+Note that `startActive(true)` finishes the span on `Scope.close()`.
+Use it carefully because the `try-with-resources` construct finishes the span before
+the `catch` or `finally` blocks are executed, which makes logging exceptions and 
+setting tags impossible. It is recommended to start the span and activate it later in `try-with-resources`.
+This makes the span available in catch and finally blocks.
 
 ```java
 io.opentracing.Tracer tracer = ...;
 ...
-Scope scope = tracer.buildSpan("someWork").startActive();
-try {
+Span span = tracer.buildSpan("someWork").start();
+try (Scope scope = tracer.scopeManager().activate(span, false))
     // Do things.
+} catch {
+    Tags.ERROR.set(scope.span(), true);
 } finally {
-    scope.deactivate();
+    span.finish();
 }
 ```
 
-To manually step around the `ScopeManager` registration, use `startManual()`, like this:
+The following code uses `try-with-resources` to finish the span.
 
 ```java
 io.opentracing.Tracer tracer = ...;
 ...
-Span span = tracer.buildSpan("someWork").startManual();
-try {
-    // (do things / record data to `span`)
-} finally {
-    span.finish();
+try (Scope scope = tracer.buildSpan("someWork").startActive(true)) {
+    // Do things.
+    //
+    // `scope.span()` allows us to pass the `Span` to newly created threads.
 }
```

Comment:
might be good to add a catch clause with a comment 

`// cannot record the error in the span since scope is not accessible`